src/plugins/embeddable/public/plugin.tsx

```diff
@@ -125,9 +125,13 @@ export class EmbeddablePublicPlugin implements Plugin<EmbeddableSetup, Embeddabl
 
   constructor(initializerContext: PluginInitializerContext) {}
 
-  public setup(core: CoreSetup, { uiActions }: EmbeddableSetupDependencies) {
+  public async setup(core: CoreSetup, { uiActions }: EmbeddableSetupDependencies) {
     bootstrap(uiActions);
 
+    const { ErrorEmbeddableFactory } = await import('./lib/embeddables/error_embeddable_factory');
```

Comment:
This doesn't seem to be right 🤔 
1. afaik platform team plans to make these handlers synchronous 
2. this is still on practice falls into "initial bundles" because setup is called in the very beginning. It would also block other plugins from initialising until this loads. 

If there is no other way to code split it, then on practice it makes more sense to leave regular input even though metrics tell otherwise.

Reply:
Yes, seems like it has to remain synchronous for now. I have updated the build metrics.